### ISSUE TITLE
8.2.0: AclObject's inherit_flags is marked as list[bool] fails on type is wanted is str

### ISSUE THREAD

**Issue by sjorge** [Isilon/isilon_sdk_8_0_python#18]
Not the pretties code but
```
                AclObject(
                    accesstype = "allow",
                    accessrights = [
                        "dir_gen_read",
                        "dir_gen_write",
                        "dir_gen_execute",
                        "std_delete",
                    ],
                    inherit_flags = [True, True],
                    trustee = dirperm.group,
                ),
```

PAPI returnes the following error:
```
{
"errors" :
[

{
"code" : "AEC_BAD_REQUEST",
"message" : "flags is not a valid string true\n."
}
]
}
```

If I specify `['true', 'true']` it no longer errors but it doesn't apply it to the actually ACL.

**Comment by sjorge** [comment]
Here is a better example... this is a dir with a simple ACL where Domain Users has modify and nothing else very special.

```
            curacl =  api.get_acl("ifs/smb/testsmb_s10", True)
            print(curacl)
            api.set_acl("ifs/smb/testsmb_s10", True, curacl)
```

As you can see below the value for `inherit_flags` is indeed a list of 2 bools but we get an error back, I also tried 2 strings using 'true' and 'True' with do not result in an error but, they do not properly set the inherit flags either :(

```
{'acl': [{'accessrights': ['dir_gen_read',
                           'dir_gen_write',
                           'dir_gen_execute',
                           'std_delete'],
          'accesstype': 'allow',
          'inherit_flags': [True, True],
          'op': None,
          'trustee': {'id': 'SID:S-1-5-21-4030456262-320625612-449655040-513',
                      'name': 'UGENT\\domain users',
                      'type': 'group'}}],
 'authoritative': 'acl',
 'group': {'id': 'SID:S-1-5-21-4030456262-320625612-449655040-513',
           'name': 'UGENT\\domain users',
           'type': 'group'},
 'mode': '0770',
 'owner': {'id': 'SID:S-1-5-18', 'name': 'SYSTEM', 'type': 'wellknown'}}
Traceback (most recent call last):
  File "/Users/jorschra/Documents/repos/isi_ugent/bin/ism.py", line 17, in <module>
    ism.run()
  File "/Users/jorschra/Documents/repos/isi_ugent/lib/ugent/isi/cli.py", line 83, in run
    ret_state, ret_failed = mgr.apply()
  File "/Users/jorschra/Documents/repos/isi_ugent/lib/ugent/isi/manager/export.py", line 106, in apply
    api.set_acl("ifs/smb/testsmb_s10", True, curacl)
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/api/namespace_api.py", line 2298, in set_acl
    (data) = self.set_acl_with_http_info(namespace_path, acl, namespace_acl, **kwargs)  # noqa: E501
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/api/namespace_api.py", line 2393, in set_acl_with_http_info
    collection_formats=collection_formats)
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/api_client.py", line 346, in call_api
    _preload_content, _request_timeout)
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/api_client.py", line 177, in __call_api
    _request_timeout=_request_timeout)
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/api_client.py", line 397, in request
    body=body)
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/rest.py", line 369, in PUT
    body=body)
  File "/Users/jorschra/Documents/repos/isi_ugent/venv/lib/python2.7/site-packages/isi_sdk_8_2_0/rest.py", line 236, in request
    raise ApiException(http_resp=r)
isi_sdk_8_2_0.rest.ApiException: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'x-isi-ifs-spec-version': '1.0', 'Content-Security-Policy': "default-src 'none'", 'Transfer-Encoding': 'chunked', 'Strict-Transport-Security': 'max-age=31536000;', 'Server': 'Apache', 'Connection': 'close', 'Allow': 'GET, PUT, POST, DELETE, HEAD', 'Date': 'Thu, 14 Nov 2019 08:41:33 GMT', 'X-Frame-Options': 'sameorigin', 'Content-Type': 'application/json'})
HTTP response body:
{
"errors" :
[

{
"code" : "AEC_BAD_REQUEST",
"message" : "flags is not a valid string true\n."
}
]
}
```

**Comment by nalinkanwar** [comment]
Fixed in v8.2.1 branch (Isilon SDK release 0.2.8)
